src/checks/y_check_profile_message.clas.abap

```diff
@@ -10,11 +10,13 @@ CLASS y_check_profile_message DEFINITION PUBLIC INHERITING FROM y_check_base CRE
   PRIVATE SECTION.
     CLASS-DATA ran TYPE abap_bool.
     METHODS get_profiles RETURNING VALUE(result) TYPE y_if_profile_manager=>profile_assignments.
-    METHODS list_profiles IMPORTING profiles TYPE y_if_profile_manager=>profile_assignments
-                          RETURNING value(result) TYPE string.
+    METHODS list_profiles IMPORTING profiles      TYPE y_if_profile_manager=>profile_assignments
+                          RETURNING VALUE(result) TYPE string.
 ENDCLASS.
 
-CLASS y_check_profile_message IMPLEMENTATION.
+
+
+CLASS Y_CHECK_PROFILE_MESSAGE IMPLEMENTATION.
```

Comment:
https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#use-your-pretty-printer-team-settings

Only keywords should be uppercase. It applies to all of the objects changed by this pull request.